nexus/translations/translations.yaml

```diff
@@ -752,3 +752,146 @@ ru:
   UPLOADED_TO_TELEGRAM: загружено в Телеграм...
   VIEWS_CANNOT_BE_SHARED: Выполните поиск сами, команду нельзя передавать другому пользователю.
   YEAR: Год
+  ua:
```

Comment:
Here you should remove the align, `ua` must goes from the very beginning of the line